src/SDDPoptimize.jl

```diff
@@ -470,7 +469,7 @@ Return:
     Vector{Float64}: optimal control at time t
 
 """
-function get_control(model::SPModel, param::SDDPparameters, lpproblem::Vector{JuMP.Model}, t, xt, xi)
+function get_control(model::SPModel, param::SDDPparameters, lpproblem::Vector{JuMP.Model}, t::Int, xt::Vector{Float64}, xi::Vector{Float64})
```

Comment:
`Int` is an alias for `Int64` or `Int32` depending on the machine. Unless you strictly need an `Int64` it is better to use `Int`.